sqlmesh/core/snapshot/evaluator.py

```diff
@@ -90,11 +90,18 @@ class SnapshotEvaluator:
         adapter: The adapter that interfaces with the execution engine.
         ddl_concurrent_tasks: The number of concurrent tasks used for DDL
             operations (table / view creation, deletion, etc). Default: 1.
+        engine_adapters: Optional dictionary of multiple engine adapters.
     """
 
-    def __init__(self, adapter: EngineAdapter, ddl_concurrent_tasks: int = 1):
+    def __init__(
+        self,
+        adapter: EngineAdapter,
```

Comment:
can we get rid of adapter and instead make it

adapters: EngineAdapter | t.Dict[str, EngineAdapter]

that way there's only one property to deal with